dff/script/extras/conditions/models/remote_api/hf_api_model.py

```diff
@@ -0,0 +1,134 @@
+"""
+HuggingFace API Model
+---------------------------
+
+This module provides the :py:class:`~HFAPIModel` class that allows you
+to use remotely hosted HuggingFace models via the HuggingFace inference API.
+"""
+import time
+import json
+import asyncio
+from typing import Optional
+from urllib.parse import urljoin
+
+import requests
```

Comment:
`requests` package is not always required by our framework, is it?
I think it should be moved inside `try ... except` block.